### DISCUSSION TITLE
I am not able to access admin tools in easyengine v4, showing 403 error

### DISCUSSION THREAD

**Discussion by oznisosa** [EasyEngine/easyengine#1619] [Support Requests]
Hi, I have a WordPress website on Easy Engine v4. I am trying to access the admin tools but I get a 403 Forbidden error.

I have successfully enabled the admin tools and have access to the global authentication password successfully. Even after entering the correct username and password for the admin tool, it shows a 403 error.

How can i solve this problem. I read several articles, but they all suggest enabling the admin tools and nothing else.

I have already enabled admin tools via ee enable admin-tools command and have access to username and password.

Please help me with this, I strongly ask.

Thanks in advance.

Greetings and blessings and best wishes.

**Comment by rw-g** [comment]
Bumping. I have the same issue.  My ee admin credentials are accepted but then I get the 403 error.

**Comment by og-spirit** [comment]
Was this ever resolved? i am having the same issue.

**Reply by dolezalll** [reply]
It's also been happening to me for the last 2 months

**Comment by marcoandrei** [comment]
Friends, did you activate the Admin Tools in the site too? (https://easyengine.io/handbook/admin-tools/)
